xwiki-platform-core/xwiki-platform-oldcore/src/main/java/org/xwiki/store/TemporaryAttachmentSessionsManager.java

```diff
@@ -139,17 +187,17 @@ default Optional<XWikiAttachment> getUploadedAttachment(AttachmentReference atta
      *
      * @param documentReference the reference for which to retrieve the temporary attachments.
      * @return {@code true} if there was some temporary attachments in cache for the given document reference in the
-     *          current user session, {@code false} if there was no matching temporary attachment in cache.
+     *         current user session, {@code false} if there was no matching temporary attachment in cache.
      */
     boolean removeUploadedAttachments(DocumentReference documentReference);
 
     /**
      * This method aims at attaching the {@link XWikiAttachment} that might have been previously temporary upload to the
-     * {@link XWikiDocument} they are targeting.
-     * This method should only be called before performing a save of the document to actually persist them. Also note
-     * that this method cannot call {@link #removeUploadedAttachment(DocumentReference, String)} as removing the
-     * attachment would delete the data before they can be properly saved in the persistent storage. So the consumer of
-     * the API should take care of properly calling this API when needed.
+     * {@link XWikiDocument} they are targeting. This method should only be called before performing a save of the
+     * document to actually persist them. Also note that this method cannot call
+     * {@link #removeUploadedAttachment(DocumentReference, String)} as removing the attachment would delete the data
+     * before they can be properly saved in the persistent storage. So the consumer of the API should take care of
+     * properly calling this API when needed.
```

Comment:
It would be nice if we could avoid this reformatting that just makes the diff larger and the PR harder to review.

Reply:
Adding a `<p>` for the auto formatting to behave as initially intended (at least it seems to be the initial intent).